test/vision/videoIndexerV2Test.js

```diff
@@ -0,0 +1,118 @@
+const cognitive = require('../../src/index.js');
+const config = require('../config.js');
+const should = require('should');
+
+describe('Video Indexer V2', () => {
+
+    const client = new cognitive.videoIndexerV2({
+        apiKey: config.videoIndexerV2.apiKey
+    });
+
+    const videoUrl = 'https://mparnisari.blob.core.windows.net/storage/video_girl_laughing.mp4';
+    const videoPath = './test/assets/video_girl_laughing.mp4';
+
+    describe('Get Accounts', () => {
```

Comment:
I don't like having several `describe`s in one file. It makes it harder to run just once suite of tests. If the Auth API is specific to the Video Indexer API, please only have one `describe`. Else, move them to a separate test file.